docs/HANDLERS.md

```diff
@@ -0,0 +1,376 @@
+# Handlers
+
+In addition to the Applications the toolkit provides a variety of the utilities to perform most common actions of the ship. Following list of handlers includes a prebuild Express routers which performs most common operations of connectors. 
+
+### actionHandler() `incoming`
+This is the simplest requests handler to expose custom logic through custom API POST endpoint. The possible usage is triggering a custom operation (like fetching historical data) or a webhook. Both cases handle data `incoming` into Hull platform. However in case of busy webhook it's better to use `batcherHandler` which automatically group the incoming requests into batches.
+
+```js
+const app = new Hull.App({}).server();
+import { actionHandler } from "hull/lib/utils";
+
+app.use("/fetch-all", actionHandler((ctx, { query, body }) => {
+  const { hull, ship } = ctx;
```

Comment:
The idea here is that the developer is not forced to use the `actionHandler`, he can use it if he finds it more useful. Nothing stops him from doing:
`app.post("/fetch-all", (req, res) => {});`

To make it clear I will add appropriate examples.